types/redux-mock-store/v0/index.d.ts

```diff
@@ -1,12 +1,14 @@
 import * as Redux from "redux";
 
-export interface MockStore<T> extends Redux.Store<T> {
-    getActions(): any[];
-    clearActions(): void;
-}
+declare namespace reduxMockStore {
+    type MockStore<T> = Redux.Store<T>;
+    type MockStoreCreator<T = {}> = (state?: T) => MockStore<T>;
 
-export type MockStoreCreator<T = {}> = (state?: T) => MockStore<T>;
+    interface ConfigureStore {
+        <T>(middlewares?: Redux.Middleware[]): MockStoreCreator<T>;
+    }
+}
 
-declare function createMockStore<T>(middlewares?: Redux.Middleware[]): MockStoreCreator<T>;
+declare const reduxMockStore: reduxMockStore.ConfigureStore;
 
-export default createMockStore;
+export = reduxMockStore;
```

Comment:
ATTW says:

> The JavaScript appears to set both module.exports and module.exports.default for improved compatibility, but the types only reflect the latter (by using export default). This will cause TypeScript under the node16 module mode to think an extra .default property access is required, which will work at runtime but is not necessary. These types export = an object with a default property instead of using export default.

Which implies that there should be an exported property called `default` in addition to the `export =`.

Reply:
https://arethetypeswrong.github.io/?p=%40types%2Fredux-mock-store%400.0.18

Reply:
Er, no, sorry, it is just talking about the types themselves as they are now.